extensions/types/vscode-mssql.d.ts

```diff
@@ -228,7 +228,8 @@ declare module 'vscode-mssql' {
 		SqlDataWarehouse = 6,
 		SqlStretchDatabase = 7,
 		SqlManagedInstance = 8,
-		SqlOnDemand = 11
+		SqlOnDemand = 11,
+		SqlDbFabric = 12
```

Comment:
make sure to also update this in the vscode-mssql repo https://github.com/microsoft/vscode-mssql/blob/main/typings/vscode-mssql.d.ts#L221

Reply:
create the PR: https://github.com/microsoft/vscode-mssql/pull/17947